modules/templates-writing-parameters.adoc

```diff
@@ -63,13 +63,19 @@ characters long consisting of all upper and lowercase alphabet letters
 and numbers.
 
 The syntax available is not a full regular expression syntax. However, you can
-use `\w`, `\d`, and `\a` modifiers:
+use `\\w`, `\\d`, `\\a`, and `\\A` modifiers:
 
-- `[\w]{10}` produces 10 alphabet characters, numbers, and underscores. This
+- `[\\w]{10}` produces 10 alphabet characters, numbers, and underscores. This
 follows the PCRE standard and is equal to `[a-zA-Z0-9_]{10}`.
-- `[\d]{10}` produces 10 numbers. This is equal to `[0-9]{10}`.
-- `[\a]{10}` produces 10 alphabetical characters. This is equal to
+- `[\\d]{10}` produces 10 numbers. This is equal to `[0-9]{10}`.
+- `[\\a]{10}` produces 10 alphabetical characters. This is equal to
 `[a-zA-Z]{10}`.
+- `[\\A]{10}` produces 10 punctuation or symbol characters. This is equal to `[~!@#$%\^&*()\-_+={}\[\]\\|<,>.?/"';:]{10}`.
+
+[NOTE]
```

Comment:
Just a side note, if you add a note specific to this last bullet for `[\A]{10}`, you can add a `+` above the note so that it continues at the same indentation as the bullet point.